build/Microsoft.DotNet.Cli.Monikers.props

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <SdkBrandName>Microsoft .NET Core 1.0.0 - SDK Preview 2</SdkBrandName>
```

Comment:
I only ask because I have no idea how to localize strings in an MSBuild file, but before we were in C#, and I know how to localize it.